src/MiniCover/Reports/BaseReport.cs

```diff
@@ -11,7 +11,7 @@ public abstract class BaseReport
         public virtual int Execute(InstrumentationResult result, float threshold)
         {
             var hits = File.Exists(result.HitsFile)
-                               ? File.ReadAllLines(result.HitsFile).Select(h => int.Parse(h)).Distinct().ToHashSet()
+                               ? File.ReadAllLines(result.HitsFile).Select(h => int.Parse(h)).ToHashSet()
```

Comment:
I added distinct because the hits file might have duplicated lines and I was afraid ToHashSet would throw exception on that case.

But I just checked and it doesn't throw exceptions. Good job!